x/wasm/keeper/wasmtesting/mock_engine.go

```diff
@@ -109,6 +111,14 @@ func (m *MockWasmer) Execute(codeID wasmvm.Checksum, env wasmvmtypes.Env, info w
 	return m.ExecuteFn(codeID, env, info, executeMsg, store, goapi, querier, gasMeter, gasLimit, deserCost)
 }
 
+func (m *MockWasmer) CallCallablePoint(name []byte, checksum wasmvm.Checksum, isReadonly bool, callstack []byte, env wasmvmtypes.Env, argsEv []byte, store wasmvm.KVStore, goapi wasmvm.GoAPI, querier wasmvm.Querier, gasMeter wasmvm.GasMeter, gasLimit uint64, deserCost wasmvmtypes.UFraction) ([]byte, wasmvmtypes.Events, wasmvmtypes.EventAttributes, uint64, error) {
+	if m.CallCallablePointFn == nil {
+		panic("not supposed to be called!")
+	}
+	fmt.Println("MockWasmer CallCallablePoint Called")
```

Comment:
Is this necessary?
I didn't think it was necessary because no other function had it.

Reply:
Thank you. Unnecessary code. I forgot to delete it so I deleted it.